playwright/e2e/crypto/crypto.spec.ts

```diff
@@ -392,7 +392,7 @@ test.describe("Cryptography", function () {
             await bobSecondDevice.sendMessage(testRoomId, "test encrypted from unverified");
             await expect(lastTile).toContainText("test encrypted from unverified");
             await expect(lastTileE2eIcon).toHaveClass(/mx_EventTile_e2eIcon_warning/);
-            await lastTileE2eIcon.focus();
+            await lastTileE2eIcon.hover();
```

Comment:
Why does focus not work anymore? This implies that it'll break with Screenreader shortcuts as they cause programmatic focus.

Reply:
Reverted, found the issue https://github.com/element-hq/compound-web/pull/159